src/views/text_area.rs

```diff
@@ -362,6 +362,11 @@ impl TextArea {
 }
 
 impl View for TextArea {
+    fn required_size(&mut self, constraint: Vec2) -> Vec2 {
+        self.compute_rows(constraint);
+        Vec2::new(1, self.rows.len())
+    }
```

Comment:
I fixed this, but that doesn't really work as expected. The problem is that `compute_rows` computes the rows dependent on the `constraint`, so it may wrap unnecessarily.

Should I pass something like `Vec2::new(MAXSIZE, MAXSIZE)` to `compute_rows` to fix this or write my own linecounting?